perception_dataset/t4_dataset/annotation_files_generator.py

```diff
@@ -110,18 +120,19 @@ def convert_one_scene(
                     cam_idx = self._camera2idx[cam]
 
                     frame_index = int((sample_nuim["filename"].split("/")[2]).split(".")[0])
-                    frame_index_to_sample_data_token[cam_idx].update(
-                        {frame_index: sample_nuim["token"]}
-                    )
-
-                    width: int = sample_nuim["width"]
-                    height: int = sample_nuim["height"]
-                    object_mask: NDArray = np.array(
-                        [[0 for _ in range(height)] for __ in range(width)], dtype=np.uint8
-                    )
-                    object_mask = cocomask.encode(np.asfortranarray(object_mask))
-                    object_mask["counts"] = repr(base64.b64encode(object_mask["counts"]))[2:]
-                    mask[cam_idx].update({frame_index: object_mask})
+                    if cam_idx < num_cameras:
```

Comment:
Why should we add this line?

Reply:
As I noted above, currently, the number of cameras are hard coded by 6 in the above line.
Then if the `camera_index` has the number of cameras greater than 6 it raises error.

Reply:
In https://github.com/tier4/tier4_perception_dataset/pull/164/commits/c11afdd06d5c557622462f79e17e6892568c514e, I fixed to avoid to use `if cam_idx < num_cameras`.